torch/ao/quantization/quantizer/xnnpack_quantizer_utils.py

```diff
@@ -344,21 +346,11 @@ def _do_annotate_conv_relu(
             continue
         relu_node = n
         maybe_conv_node = n.args[0]
-        # TODO: refactor with is_conv_node and is_conv_transpose_node
-        if is_conv_transpose:
-            conv_ops = [
-                torch.ops.aten.conv_transpose1d,
-                torch.ops.aten.conv_transpose2d.input,
-            ]
-        else:
-            conv_ops = [
-                torch.ops.aten.conv1d.default,
-                torch.ops.aten.conv2d.default,
-            ]
+
         if (
             not isinstance(maybe_conv_node, Node)
-            or maybe_conv_node.op != "call_function"
-            or maybe_conv_node.target not in conv_ops
+            or (is_conv_transpose and not _is_conv_transpose_node(maybe_conv_node))
+            or (not is_conv_transpose and not _is_conv_node(maybe_conv_node))
```

Comment:
this looks a bit complicated, can this be simplified? maybe remove the `is_conv_transpose` flag and just pass in a function to check the node?